webapp-django/crashstats/crashstats/utils.py

```diff
@@ -54,6 +54,11 @@ def wrapper(request, *args, **kw):
         response = f(request, *args, **kw)
 
         if isinstance(response, http.HttpResponse):
+            if (
+                isinstance(response, http.HttpResponseBadRequest) and
+                not response['Content-Type'].startswith('application/json')
+            ):
+                response['Content-Type'] = 'application/json; charset=UTF-8'
```

Comment:
Also, what happens here if the content actually isn't JSON? Then the content and content-type don't match.